### DISCUSSION TITLE
開版首帖

### DISCUSSION THREAD

**Discussion by awesome-doge** [cypherpunks-core/cypherpunks-core.github.io#51] [General]
歡迎大家 加入 密碼龐克 論壇呀
* 討論技術 加密貨幣技術, 去中心化技術, IPFS, Tor , Signal 都可以
* 歡迎大家執行任何專案
*  特殊專案 寫書、衛星計畫
* 一起在 Taipei Hacker Space 吃火鍋 娛樂 吃飯 喝酒 看電影 出去玩

**Comment by aabb0404** [comment]
一起改車?

**Reply by awesome-doge** [reply]
改爆 昨天才去
[地點](https://www.google.com/maps/place/Exquisite-Power(%E7%B2%BE%E7%B7%BB%E5%8B%95%E5%8A%9B)/@25.0612793,121.6122653,20.34z/data=!4m12!1m6!3m5!1s0x0:0xe9c6fa14e2b9242a!2sBitmark+Inc.!8m2!3d25.060971!4d121.611905!3m4!1s0x3442aca6c4f5f4eb:0xd587028e05d3a724!8m2!3d25.061096!4d121.6120387)

我昨天才去換輪胎 直接上 登祿普TT93 競技胎  超Ｑ  超黏 超避震  一開始 還會狂黏沙子
價格一千五
![image](https://user-images.githubusercontent.com/54278075/110075721-3a483100-7dbe-11eb-9d8e-575dd3b44492.png)

**Reply by aabb0404** [reply]
壓車 不會分離 你有問他多久要清理嗎?

**Reply by milochen0418** [reply]
# 馬的
## 超高級
### 讚

**Reply by antonim63** [reply]
Antonim63

**Comment by milochen0418** [comment]
# 到此一遊

**Reply by awesome-doge** [reply]
歡迎

**Comment by mcu0054** [comment]
如果商業用戶想要在區塊鏈儲存跟搜尋資料,像是區塊鏈食品履歷那樣
以成本來說是不是
Hyperledger (Fabric) > 私有鏈  >公有鏈
然後以安全性來說
公有鏈>Hyperledger (Fabric) > 私有鏈

**Reply by awesome-doge** [reply]
我覺得主要會是 客製化服務 
我覺得真的公有鏈 是沒有需要的  雖然安全等級第一  但其實這些客製化的數據 根本就沒有必要上到公有鏈上 保存
論客製化服務 Hyperledger 有很多的系列 我覺得可以找一下你需要的是哪個版本 裡面應該有你要的

**Comment by ru19wu06** [comment]
Cypherpunks 應該約個一段時間聚一次，感覺會不錯

**Reply by awesome-doge** [reply]
阿你不是昨天才聚過

**Reply by ru19wu06** [reply]
要更多人，更嗨?

**Comment by ru19wu06** [comment]
我覺得可以辦個Cypherpunks Private party，切合Cypherpunk的主題XD

**Reply by awesome-doge** [reply]
所以什麼時候來辦活動？

**Reply by ru19wu06** [reply]
> 所以什麼時候來辦活動？

你登高一呼我就去幫忙ㄌ

**Reply by milochen0418** [reply]
@ru19wu06  你有沒有想要主辦什麼活動呀?
